### ISSUE TITLE
Fix request for false positive in UnconditionalConcatInLogArgument

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#141]
False positive:
`log.info("Execution Time : {} ms for {} no of calls", endTime - startTime, size);`

The '-' is confused with '+'

**Comment by jborgers** [comment]
Test cases:

```
@slf4j
class Foo {
    void bar() {
         long startTime = System.currentTimeMillis();
         long endTime = System.currentTimeMillis();
         log.info("Execution Time : {} ms ", endTime - startTime); // good, no concat
         log.info("More Time : {} ms ", endTime + startTime); // good, no concat
         String strStart = String.valueOf(startTime);
         String strEnd = String.valueOf(endTime);
         log.info("Concatenation of Strings {} ", strEnd + strStart); // bad
         log.info("Concatenation of String and converted value {} ", strEnd + startTime); // bad
         log.info("Concatenation of converted value and String {} ", endTime + strStart); // bad
         log.info("Concat confusing : {} ms ", "" + endTime + startTime); // bad
    }
}
```
